browser/src/Query.tsx

```diff
@@ -37,17 +37,16 @@ const cancelable = (promise: any) => {
   }
 }
 
-type OwnBaseQueryProps = {
+type BaseQueryState = any
```

Comment:
Just for my curiosity and TS education, is the state of a generic `Component<Props, State>` often hard to concretely type, thus `any` is an acceptable type for it? 

I'm curious as to the reasoning behind the `any` here. I have no clue what it would be otherwise, and wondered if you could enlighten me a bit.

Reply:
In this case, the `any` was inserted automatically by `ts-migrate` a while back, presumably because it wasn't able to figure out a more specific type. States usually are just a typical JS/TS object, nothing exotic, so not especially difficult to write types for. Just haven't touched this one in particular yet.

Reply:
Oh oh right oops. I didn't realize this was Git's diff looking slightly funky (removing the line in one place, and adding it in another place). I see, that makes sense.